athenian/database_pg.go

```diff
@@ -223,6 +224,85 @@ func (db *database) AthenianToGithub(ctx context.Context, id AccountID) ([]Githu
 	return out, rows.Err()
 }
 
+func (db *database) GetInstaflowStatus(ctx context.Context, accID AccountID) (*InstaflowStatus, error) {
+	return db.getInstaflowStatus(ctx, accID)
+}
+
+func (db *database) getInstaflowStatus(ctx context.Context, accID AccountID) (*InstaflowStatus, error) {
+	row := db.db.QueryRow(
+		ctx,
+		`SELECT account_id, account_created, fetch_started, fetch_completed, consistency_started, consistency_completed, precompute_started, precompute_completed, current_status
+		FROM public.installation_progress WHERE account_id = $1`,
+		int64(accID),
+	)
+	instaflowStatus, err := scanInstaflowStatus(row)
+	if err == pgx.ErrNoRows {
+		err = ErrNotFound
+	}
+	return &instaflowStatus, err
+}
+
+func scanInstaflowStatus(sc dbs.Scanner) (InstaflowStatus, error) {
+	var (
+		accID                int64
+		accountCreated       time.Time
+		fetchStarted         time.Time
+		fetchCompleted       time.Time
+		consistencyStarted   time.Time
+		consistencyCompleted time.Time
+		precomputeStarted    time.Time
+		precomputeCompleted  time.Time
+		status               string
+	)
+	err := sc.Scan(&accID, &accountCreated, &fetchStarted, &fetchCompleted, &consistencyStarted, &consistencyCompleted, &precomputeStarted, &precomputeCompleted, &status)
+	if err == pgx.ErrNoRows {
+		err = ErrNotFound
+	}
+
+	return InstaflowStatus{
+		AccID:                AccountID(accID),
+		AccountCreated:       accountCreated,
+		FetchStarted:         fetchStarted,
+		FetchCompleted:       fetchCompleted,
+		ConsistencyStarted:   consistencyStarted,
+		ConsistencyCompleted: consistencyCompleted,
+		PrecomputeStarted:    precomputeStarted,
+		PrecomputeCompleted:  precomputeCompleted,
```

Comment:
better to convert them to UTC already here

Reply:
make sense